### PR TITLE
Refactor reconcile and resume cmd for alert and receiver

### PR DESCRIPTION
**Pull request by SomtochiAma**
Part of: #291 

Signed-off-by: Somtochi Onyekwere <somtochionyekwere@gmail.com>

### CODE REVIEW

File: cmd/flux/reconcile_kustomization.go [hunk 1]
```diff
@@ -17,22 +17,9 @@ limitations under the License.
...
-
-	"github.com/fluxcd/flux2/internal/utils"
-	"github.com/fluxcd/pkg/apis/meta"
-	"github.com/spf13/cobra"
-	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/apimachinery/pkg/util/wait"
-
 	kustomizev1 "github.com/fluxcd/kustomize-controller/api/v1beta1"
 	sourcev1 "github.com/fluxcd/source-controller/api/v1beta1"
+	"github.com/spf13/cobra"
```
Comment (hiddeco) [inline cmd/flux/reconcile_kustomization.go:22]:
```suggestion
	"github.com/spf13/cobra"

	kustomizev1 "github.com/fluxcd/kustomize-controller/api/v1beta1"
	sourcev1 "github.com/fluxcd/source-controller/api/v1beta1"
```

File: cmd/flux/reconcile_with_source.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+import (
+	"context"
+	"fmt"
+	"github.com/fluxcd/flux2/internal/utils"
+	"github.com/fluxcd/pkg/apis/meta"
+	"github.com/spf13/cobra"
+	apimeta "k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
```
Comment (hiddeco) [inline cmd/flux/reconcile_with_source.go:11]:
```suggestion
	"context"
	"fmt"

	"github.com/spf13/cobra"
	apimeta "k8s.io/apimachinery/pkg/api/meta"
	"k8s.io/apimachinery/pkg/types"
	"k8s.io/apimachinery/pkg/util/wait"
	
	"github.com/fluxcd/pkg/apis/meta"
	
	"github.com/fluxcd/flux2/internal/utils"
```

File: cmd/flux/resume_alert.go [hunk 3]
```diff
@@ -17,20 +17,9 @@ limitations under the License.
...
 
 import (
-	"context"
-	"fmt"
-
-	"github.com/fluxcd/flux2/internal/utils"
-	"github.com/fluxcd/pkg/apis/meta"
+	notificationv1 "github.com/fluxcd/notification-controller/api/v1beta1"
 
 	"github.com/spf13/cobra"
```
Comment (hiddeco) [inline cmd/flux/resume_alert.go:22]:
```suggestion
	"github.com/spf13/cobra"

	notificationv1 "github.com/fluxcd/notification-controller/api/v1beta1"
```

File: cmd/flux/resume_receiver.go [hunk 4]
```diff
@@ -17,20 +17,8 @@ limitations under the License.
...
-
-	"github.com/spf13/cobra"
-	apimeta "k8s.io/apimachinery/pkg/api/meta"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/apimachinery/pkg/util/wait"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-
 	notificationv1 "github.com/fluxcd/notification-controller/api/v1beta1"
+	"github.com/spf13/cobra"
```
Comment (hiddeco) [inline cmd/flux/resume_receiver.go:21]:
```suggestion
	"github.com/spf13/cobra"
	
	notificationv1 "github.com/fluxcd/notification-controller/api/v1beta1"
```

File: cmd/flux/suspend_receiver.go [hunk 5]
```diff
@@ -17,14 +17,8 @@ limitations under the License.
...
 import (
-	"context"
-	"fmt"
-
-	"github.com/spf13/cobra"
-	"k8s.io/apimachinery/pkg/types"
-
-	"github.com/fluxcd/flux2/internal/utils"
 	notificationv1 "github.com/fluxcd/notification-controller/api/v1beta1"
+	"github.com/spf13/cobra"
```
Comment (hiddeco) [inline cmd/flux/suspend_receiver.go:21]:
```suggestion
	"github.com/spf13/cobra"

	notificationv1 "github.com/fluxcd/notification-controller/api/v1beta1"
```

File: cmd/flux/reconcile_helmrelease.go [hunk 6]
```diff
@@ -17,23 +17,9 @@ limitations under the License.
...
-	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/client-go/util/retry"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-
-	"github.com/fluxcd/flux2/internal/utils"
-	"github.com/fluxcd/pkg/apis/meta"
-
 	helmv2 "github.com/fluxcd/helm-controller/api/v2beta1"
 	sourcev1 "github.com/fluxcd/source-controller/api/v1beta1"
+	"github.com/spf13/cobra"
```
Comment (hiddeco) [inline cmd/flux/reconcile_helmrelease.go:22]:
```suggestion
	"github.com/spf13/cobra"

	helmv2 "github.com/fluxcd/helm-controller/api/v2beta1"
	sourcev1 "github.com/fluxcd/source-controller/api/v1beta1"
```

File: cmd/flux/reconcile.go [hunk 7]
```diff
@@ -19,6 +19,7 @@ package main
 import (
 	"context"
 	"fmt"
+	"github.com/fluxcd/notification-controller/api/v1beta1"
 	"time"
```
Comment (hiddeco) [inline cmd/flux/reconcile.go:23]:
```suggestion
	"context"
	"fmt"
	"time"
	
	"github.com/fluxcd/notification-controller/api/v1beta1"
```

**Review by hiddeco — COMMENTED** [review]
Think the code content looks OK, but the order of imports is wrong, we group them to make it easier to read:

1. builtin Go packages
2. external packages
3. external `fluxcd` packages
4. internal `fluxcd/flux2` packages

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @SomtochiAma 🏅

**Comment by stefanprodan** [comment]
> Think the code content looks OK, but the order of imports is wrong, we group them to make it easier to read

@hiddeco all of @SomtochiAma PRs have mixed imports, I think it's best to fix them for all files in a different PR.

**Comment by hiddeco** [comment]
SGTM.

**Pull request merged**